model/project_aliases.go

```diff
@@ -96,13 +104,48 @@ func findAliasInProject(projectID, alias string) ([]ProjectAlias, error) {
 	if err != nil {
 		return []ProjectAlias{}, errors.Wrap(err, "error finding project aliases")
 	}
+	parserProject, err := ParserProjectByVersion(projectID, version)
```

Comment:
Does it make sense to flip these? If it exists for parser project then we never need to query the alias collection.

Reply:
Want to make sure you saw this ^ feel free to DM for more clarification!

Reply:
Yep you're right.  I was thinking about FindAliasesForProject but here we are just looking for a single key.